src/components/QuickOpenModal.js

```diff
@@ -384,12 +382,17 @@ export class QuickOpenModal extends Component<Props, State> {
           onChange={this.onChange}
           onKeyDown={this.onKeyDown}
           handleClose={this.closeModal}
-          hasPrefix={this.hasPrefix()}
           expanded={expanded}
           selectedItemId={
             expanded && items[selectedIndex] ? items[selectedIndex].id : ""
           }
         />
+        {!symbols ||
+          (symbols.functions.length == 0 && (
```

Comment:
This `symbols.functions.length` will always fail if there's no query text, so we should add that to this conditional.

Reply:
Also, when I do add text to the input, and there are many results, `symbols.functions.length` is still 0, so maybe we should be doing a different check?